database/migrations/1647757480065_update_comment_columns.ts

```diff
@@ -0,0 +1,23 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class UpdateCommentColumns extends BaseSchema {
+  protected tableName = 'comments'
+
+  public async up() {
+    this.schema.alterTable(this.tableName, (table) => {
+      table.dropColumn('post')
+      table.dropColumn('user')
+      table.integer('post_id')
+      table.integer('user_id')
+    })
+  }
+
+  public async down() {
+    this.schema.alterTable(this.tableName, (table) => {
+      table.string('post')
+      table.string('user')
+      table.dropColumn('post_id')
+      table.dropColumn('user_id')
+    })
+  }
+}
```

Comment:
rather than adding 3 migration files, you can role back to before adding any of these migrations, delete the new files, create the proper migration, and then commit that instead. 1 migration vs 3 migrations is much easier to track